docs/Contributing/architecture/orchestration/live-queries.md

```diff
@@ -0,0 +1,71 @@
+# Live Queries Architecture
```

Comment:
Will want to either verify this guide (by someone who's worked with live queries recently) or drop it in favor of linking to human-generated docs, since otherwise we run the risk of having newer docs that are less accurate than older docs on the same topic. I think @getvictor has a solid public guide on this.

Reply:
Agreed @sharon-fdm

Reply:
Updating with a link to Victor's guide.